README.rst

```diff
@@ -35,10 +35,11 @@ Formula Dependencies
 ====================
 
 * `apache-formula <https://github.com/saltstack-formulas/apache-formula>`_
-* `mysql-formula <https://github.com/saltstack-formulas/mysql-formula>`_
+* `saltstack-mysql-formula <https://github.com/bechtoldt/saltstack-mysql-formula>`_
```

Comment:
This is not an official saltstack formula. Please use official saltstack formula under https://github.com/saltstack-formulas/ or create a request to get the formula merged into saltstack-formulas.